src/analysis/processing/qgsalgorithmrasterstackposition.cpp

```diff
@@ -301,7 +301,7 @@ int QgsRasterStackLowestPositionAlgorithm::findPosition( std::vector< std::uniqu
     }
   }
   //the ArcGIS implementation uses 1 for first position value instead of 0 as in standard c++
-  return ++lowestPosition; //therefore ++
+  return static_cast<int>( ++lowestPosition ); //therefore ++
```

Comment:
should we change return type of findPosition to size_t instead?

Reply:
not ideal since it returns -1 at line 275